docs/resources/deployment-examples/example-fargate/example_config/anchor_config.yaml

```diff
@@ -27,8 +27,10 @@ stellar:
     # `data-spring-jdbc` is the only `type` value supported for now.
     #
     data-access:
-      type: data-spring-jdbc
-      settings: data-spring-jdbc-sqlite              # The absolute location of the configuration data in this yaml file
+      type: data-spring-jdbc                      # Activate [config-spring-jdbc] module.
+      settings: data-spring-jdbc-h2 
+      #type: data-spring-jdbc
```

Comment:
Is it worth either restoring this to sqlite, or removing the commented lines, since this now working.